readme.md

```diff
@@ -287,6 +287,7 @@
 - [Deep Vision](https://github.com/kjw0612/awesome-deep-vision#readme)
 - [Open Source Society University](https://github.com/ossu/computer-science#readme)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming#readme)
+- [Empirical Software Engineering](https://github.com/dspinellis/awesome-msr#readme) - Data sets and tools for conducting empirical software engineering research.
```

Comment:
> The entry in the Awesome list should:
> * Include a short description about the project/theme of the list. **It should not describe the list itself.**

It should describe **what** `Empirical Software Engineering` is.

Reply:
I changed the description to describe the content, rather than the list.